File: Arquitectura de Software/02.PARCIAL/API_GOOGLEMAPS_RESTFULL_JAVA_GR01/API_GOOGLEMAPS_RESTFULL_JAVA_GR01/01. SERVICIO/servicio-web-eureka/src/main/java/ec/edu/monster/servicio_web_eureka/modelo/Sucursal.java
package ec.edu.monster.servicio_web_eureka.modelo;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "sucursal")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Sucursal {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id_sucursal")
    private Integer idSucursal;

    @Column(name = "vch_sucunombre", nullable = false, length = 255)
    private String nombre;

    @Column(name = "vch_sucuciudad", nullable = false, length = 255)
    private String ciudad;

    @Column(name = "vch_sucudireccion", length = 255)
    private String direccion;

    @Column(name = "latitud", nullable = false)
    private Double latitud;

    @Column(name = "longitud", nullable = false)
    private Double longitud;

    @Column(name = "int_sucucontcuenta", nullable = false)
    private Integer contadorCuenta;
}

File: Arquitectura de Software/02.PARCIAL/API_GOOGLEMAPS_RESTFULL_DOTNET_GR01/API_GOOGLEMAPS_RESTFULL_DOTNET_GR01/01. SERVICIO/EurekaSucursalesApi/Models/Sucursal.cs
namespace EurekaSucursalesApi.Models
{
    public class Sucursal
    {
        public int Id { get; set; }

        public string Nombre { get; set; } = string.Empty;

        public string Ciudad { get; set; } = string.Empty;

        public string Direccion { get; set; } = string.Empty;

        // Coincide con FLOAT / REAL en SQL Server
        public double Latitud { get; set; }

        public double Longitud { get; set; }
    }
}
